### DISCUSSION TITLE
Feedback: Installed kernel linux-rockchip-5.10.198-1 -- Works Fine on Existing Image

### DISCUSSION THREAD

**Discussion by JFLim1** [Joshua-Riek/ubuntu-rockchip#551] [General]
Hi @Joshua-Riek, 
Just a feedback.  Downloaded https://github.com/Joshua-Riek/ubuntu-rockchip/actions/runs/7380708499/artifacts/1142301529 (kernel-5.10.198-) and Installed installed on existing Ubuntu-22.04.3 image with:
`
sudo dpkg -i /home/jfl/Downloads/linux-rockchip/linux-image-5.10.198_5.10.198-1_arm64.deb /home/jfl/Downloads/linux-rockchip/linux-headers-5.10.198_5.10.198-1_arm64.deb`

Question: is it wise/ok just to install linux-image-5.10.198 and linux-headers-5.10.198 on the existing Ubuntu-22.04.3-Gnome-5.10.160-28 image?

So far seems to work well and stable.  Chromium streaming with Youtube with vpu hw acceleration works.  LibreOffice, Thourium-Browser as expected so far albeit just with 2:30 hrs uptime.

Any new features added to this new kernel-5.10.198-1?

**Comment by Joshua-Riek** [comment]
Thanks for the feedback. There are not really any new features from my understanding it's a security update.
